### ISSUE TITLE
Observing error on API:/v1/regard-controller-object-deletion when device connection changing to connecting 

### ISSUE THREAD

**Issue by Ziabatcha** [openBackhaul/NotificationProxy#63]
When device connection status is changing from connected to connecting, observing below error

```
[05:29:47.321] DEBUG (1): sending subscriber notification to: HTTP://x.x.x.x:xxxx/v1/regard-controller-attribute-value-change with content: {"notification-proxy-1-0:attribute-value-changed-notification":{"counter":1,"timestamp":"2023-12-14T05:29:47.312732642Z","resource":"/core-model-1-4:network-control-domain=live/control-construct=OpenDayLight1/logical-termination-point=513250004/layer-protocol=0/mount-point-1-0:mount-point-pac/mount-point-configuration","attribute-name":"connection-status","new-value":"connecting"}}
[05:29:47.361] DEBUG (1): sending subscriber notification to: HTTP://x.x.x.x:xxxx/v1/regard-controller-object-deletion with content: {"notification-proxy-1-0:object-deletion-notification":{"counter":1,"timestamp":"2023-12-14T05:29:47.312732642Z","resource":"/core-model-1-4:network-control-domain=live/control-construct=OpenDayLight1/logical-termination-point=513250004/layer-protocol=0/mount-point-1-0:mount-point-pac/mount-point-configuration/session-id"}}
[05:29:47.397] ERROR (1): error during subscriber-notification for /v1/notify-controller-object-deletions
    err: {
      "message": "Request failed with status code 404",
      "name": "AxiosError",
      "stack":
          AxiosError: Request failed with status code 404
              at settle (/home/openbackhaul/NotificationProxy/node_modules/axios/dist/node/axios.cjs:1913:12)
              at IncomingMessage.handleStreamEnd (/home/openbackhaul/NotificationProxy/node_modules/axios/dist/node/axios.cjs:2995:11)
              at IncomingMessage.emit (node:events:525:35)
              at endReadableNT (node:internal/streams/readable:1358:12)
              at processTicksAndRejections (node:internal/process/task_queues:83:21)
      "config": {
        "transitional": {
          "silentJSONParsing": true,
          "forcedJSONParsing": true,
          "clarifyTimeoutError": false
        },
        "adapter": [
          "xhr",
          "http"
        ],
        "transformRequest": [
          null
        ],
        "transformResponse": [
          null
        ],
        "timeout": 0,
        "xsrfCookieName": "XSRF-TOKEN",
        "xsrfHeaderName": "X-XSRF-TOKEN",
        "maxContentLength": -1,
        "maxBodyLength": -1,
        "env": {
          "Blob": null
        },
        "headers": {
...
...
        },
        "method": "post",
        "url": "HTTP://x.x.x.x:xxxx/v1/regard-controller-object-deletion",
        "data": "{\"notification-proxy-1-0:object-deletion-notification\":{\"counter\":1,\"timestamp\":\"2023-12-14T05:29:47.312732642Z\",\"resource\":\"/core-model-1-4:network-control-domain=live/control-construct=OpenDayLight1/logical-termination-point=513250004/layer-protocol=0/mount-point-1-0:mount-point-pac/mount-point-configuration/session-id\"}}"
      },
      "code": "ERR_BAD_REQUEST",
      "status": 404
    }
```

**Comment by pBittenbringMentopolis** [comment]
Hello Zia,
the 404-error can mean two things: The target url is not available or the resource path was not found in the MWDI.

Regarding the target url: In the current MWDI there does not exist an endpoint for "regard-controller-object-deletion" yet or do you work with another version of MWDI where it was already added?

**Comment by Ziabatcha** [comment]
Fixed as part of [v1.0.0_test-4](https://github.com/openBackhaul/NotificationProxy/releases/tag/v1.0.0_test-4), hence closing
